### PR TITLE
[v0.6.x] Adding ladon to vendor

### PR DESCRIPTION
**Pull request by bquenin**
This PR is just a sandbox to illustrate the usage of ladon and to check whether or not we want to use it for access control.

The first commit ladon vendoring.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -71,8 +71,11 @@ import:
 - package: gopkg.in/hlandau/measurable.v1
   version: ^1.0.1
 - package: github.com/holys/safe
+- package: github.com/ory-am/ladon
+  version: ^0.4.1
+- package: github.com/jmoiron/sqlx
 testImport:
 - package: github.com/stretchr/testify
-  version: ^1.1.4
+  version: 976c720a22c8eb4eb6a0b4348ad85ad12491a506
```
Comment (ndegory) [inline glide.yaml:79]: I've failed to find this commit of testify on the repo, is there any issues with tag 1.1.4?

**Review by ndegory — CHANGES_REQUESTED** [review]
vendors have been updated on master since you submitted this PR, you will have to rebase

File: glide.yaml (hunk 1 above)
Comment (bquenin) [reply glide.yaml:79]: This PR is not for master, but for branch v0.6.x. I had an issue with testify at the time, I'll have a look at it again.
Here is the testify commit I'm referring to: https://github.com/stretchr/testify/commit/976c720a22c8eb4eb6a0b4348ad85ad12491a506

**Comment by bquenin** [comment]
Closing this PR until we have a first implementation of the permission model.

**Pull request closed without merging**